src/editor/actions/sagas/eval-queue-saga.js

```diff
@@ -141,6 +89,10 @@ export function* evaluateCurrentQueue() {
       yield call(evaluateByType, chunkType, chunkContent, chunkId);
       yield put(setKernelState("KERNEL_IDLE"));
     } catch (error) {
+      if (process.env.NODE_ENV === "dev") {
```

Comment:
are we using this pattern elsewhere? If everyone's fine with it, I would recommend at least using `console.error`

Reply:
this exact pattern is new since #1948, but we've been using similar patterns for a while in a number of places. happy to switch to `console.error` if that's more appropriate